### ISSUE TITLE
[ Feature Request ] Hide instruments from a concrete pattern

### ISSUE THREAD

**Issue by Porrumentzio** [hydrogen-music/hydrogen#889]
Hydrogen version * : 1.0.0-beta2
Operating system + version : 5.4.40-1-Manjaro MATE 20.0.1, Arch based
Audio driver + version : snd_hda_intel

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
When using a pattern just to one instrument, is annoying to have them all. But as deleting an instrument in a pattern deletes it from all other patterns with the same Drumkit, this can't be done.

For that reason it would be great to be able to hide an instrument from a pattern.

**Comment by trebmuh** [comment]
You can "move" each instrument up and down in the instrument list. Then you have access to the used one without having to scroll.

**Comment by Porrumentzio** [comment]
Yes, I know I could that, but it's not the same. Also I think that, though it would be more labor of development, that feature would economize graphic resources on computers. I don't know if you understand what I mean.

Thanks for the answer, as moving instruments will ease my working :)
